jobs/workflow_test.groovy

```diff
@@ -139,13 +141,12 @@ import utilities.StatusUpdater
 
       shell """
         #!/usr/bin/env bash
-
         set -eo pipefail
-
-        ./ci.sh
-        if [ "\${COMMIT}" == "true" ]; then
-          ${defaults.bumpverCommitCmd}
-        fi
+        env > /tmp/\${JOB_NAME}/\${BUILD_NUMBER}/env.file
```

Comment:
just a note: since we don't have to interpolate any groovy/DSL-defined variables (as we did before with `${defaults.bumpverCommitCmd}`, we could change this triple-double-quoted string to a triple-single-quoted string and then we wouldn't have to escape the `$`s